skops/_min_dependencies.py

```diff
@@ -32,6 +32,7 @@
     "xgboost": ("1.6", "tests", None),
     # TODO: remove condition when catboost supports python 3.11
     "catboost": ("1.0", "tests", "python_version < '3.11'"),
+    "fairlearn": ("0.8.0", "docs, tests", None),
```

Comment:
```suggestion
    "fairlearn": ("0.7.0", "docs, tests", None),
```

This should work as well.